packages/rpc-graphql/src/loaders/__tests__/account-loader-test.ts

```diff
@@ -66,4 +66,1085 @@ describe('account loader', () => {
             expect(rpc.getAccountInfo).toHaveBeenCalledTimes(2);
         });
     });
+    describe('batch loading', () => {
```

Comment:
I didn't super double check, but do these (awesome) tests include the case where

* one thing got the default commitment of `confirmed` because it didn't specify a commitment, and
* another otherwise identical fetch explicitly stated `commitment: 'confirmed'`

We should have a test that asserts that two such requests merge into one.

Reply:
I've added this test, and unfortunately it wasn't handled! I've updated the batch loader to coalesce this.